rasgoql/requirements-postgres.txt

```diff
@@ -0,0 +1,2 @@
+SQLAlchemy
+psycopg2
```

Comment:
do we need both? i haven't installed fresh in a while, so maybe a stupid question